app/models/db/MetricsFilters.scala

```diff
@@ -66,13 +64,19 @@ object MetricsFilters {
     case "" => None
   }
 
-  def metricFilters(implicit filters: MetricsFilters): DBMetric => Rep[Option[Boolean]] = { metric =>
+  private def getOptionFromQS(key: String, qs: Map[String, Seq[String]]): Option[String] = qs.get(key).flatMap(_.headOption)
+
+  def metricFilters(implicit filters: MetricsFilters): DBMetric => Rep[Option[Boolean]] = metric => checkMetricsFilters(metric)
+
+  def forksFilters[T](implicit filters: MetricsFilters): ((T, DBMetric)) => Rep[Option[Boolean]] = forkAndMetric => checkMetricsFilters(forkAndMetric._2)
```

Comment:
It wasn't immediately obvious to me that `forkAndMetric` was the result of a join. But names are hard and I can't really think of a better one :/